src/game/overlay.c

```diff
@@ -113,40 +193,27 @@ static void Overlay_GetBarLocation(
     m_BarOffsetY[bar_location] += height + bar_spacing;
 }
 
-static void Overlay_DrawBar(int32_t value, int32_t value_max, int32_t bar_type)
+static void Overlay_DrawBar(BAR_INFO *draw_bar)
 {
-    static int32_t blink_counter = 0;
     const int32_t percent_max = 100;
 
-    if (value < 0) {
-        value = 0;
-    } else if (value > value_max) {
-        value = value_max;
+    if (draw_bar->value < 0) {
+        draw_bar->value = 0;
+    } else if (draw_bar->value > draw_bar->max_value) {
+        draw_bar->value = draw_bar->max_value;
     }
-    int32_t percent = value * 100 / value_max;
+    int32_t percent = draw_bar->value * 100 / draw_bar->max_value;
 
     const RGB888 rgb_bgnd = { 0, 0, 0 };
     const RGB888 rgb_border_light = { 128, 128, 128 };
     const RGB888 rgb_border_dark = { 64, 64, 64 };
 
     int32_t width = 100;
     int32_t height = 5;
-    int16_t bar_color = bar_type;
 
     int32_t x = 0;
     int32_t y = 0;
-    if (bar_type == BT_LARA_HEALTH) {
-        Overlay_GetBarLocation(
-            g_Config.healthbar_location, width, height, &x, &y);
-        bar_color = g_Config.healthbar_color;
-    } else if (bar_type == BT_LARA_AIR) {
-        Overlay_GetBarLocation(g_Config.airbar_location, width, height, &x, &y);
-        bar_color = g_Config.airbar_color;
-    } else if (bar_type == BT_ENEMY_HEALTH) {
-        Overlay_GetBarLocation(
-            g_Config.enemy_healthbar_location, width, height, &x, &y);
-        bar_color = g_Config.enemy_healthbar_color;
-    }
+    Overlay_GetBarLocation(draw_bar->bar_location, width, height, &x, &y);
```

Comment:
Consider making `Bar` a common prefix across all bar functions i.e. `Overlay_BarGetLocation` etc. to lump those functions together.